commit f5b46f59524cae0b2ee5dc577c75c3d2cdd76ab9
Author: Simon Quigley <tsimonq2@ubuntu.com>
Date:   Sun Jun 11 19:04:52 2017 -0500

    Revert "Automatically generate code attribution documentation"
    
    This commit needed to be reverted in order to build qtbase because the
    automatic generation tool it uses in this commit is only available in Qt
    Tools 5.8 and over, which Ubuntu does not have.
    
    This patch can be removed once Qt Tools 5.9.0 has landed in Ubuntu
    Artful.

diff --git a/doc/global/config.qdocconf b/doc/global/config.qdocconf
index a18396876d..bb7ffa8f45 100644
--- a/doc/global/config.qdocconf
+++ b/doc/global/config.qdocconf
@@ -10,7 +10,7 @@ dita.metadata.default.audience = programmer
 #Set the main Qt index.html
 navigation.homepage = "Qt $QT_VER"
 
-sourcedirs += includes $$BUILDDIR
+sourcedirs += includes
 
 url = http://doc.qt.io/qt-5
 
diff --git a/mkspecs/features/qt_docs.prf b/mkspecs/features/qt_docs.prf
index 72dde61a40..183d0c9502 100644
--- a/mkspecs/features/qt_docs.prf
+++ b/mkspecs/features/qt_docs.prf
@@ -21,16 +21,12 @@ qtvertag.name = QT_VERSION_TAG
 qtvertag.value = $$replace(qtver.value, \.,)
 qtdocs.name = QT_INSTALL_DOCS
 qtdocs.value = $$[QT_INSTALL_DOCS/src]
-builddir.name = BUILDDIR
-builddir.value = $$OUT_PWD
-QT_TOOL_ENV = qtver qtmver qtvertag qtdocs builddir
+QT_TOOL_ENV = qtver qtmver qtvertag qtdocs
 qtPrepareTool(QDOC, qdoc)
 QT_TOOL_ENV =
 
 !build_online_docs: qtPrepareTool(QHELPGENERATOR, qhelpgenerator)
 
-qtPrepareTool(QTATTRIBUTIONSSCANNER, qtattributionsscanner)
-
 # qtPrepareTool() must be called outside a build pass, as it protects
 # against concurrent wrapper creation by omitting it during build passes.
 # However, creating the actual targets is reserved to the build passes.
@@ -67,21 +63,12 @@ DOC_INDEXES =
         PREP_DOC_INDEXES += -indexdir $$shell_quote($$[QT_INSTALL_DOCS/get])
     DOC_INDEXES += -indexdir $$shell_quote($$[QT_INSTALL_DOCS/get])
 }
-
-qtattributionsscanner.target = qtattributionsscanner
-qtattributionsscanner.commands = $$QTATTRIBUTIONSSCANNER $$shell_quote($$MODULE_BASE_INDIR) \
-    --filter "QDocModule=$$QMAKE_DOCS_TARGET" -o $$shell_quote($$OUT_PWD/codeattributions.qdoc)
-qtattributionsscanner.CONFIG += phony
-QMAKE_EXTRA_TARGETS += qtattributionsscanner
-
 doc_command = $$QDOC $$QMAKE_DOCS
 prepare_docs {
     prepare_docs.commands += $$doc_command -prepare $$PREP_DOC_INDEXES -no-link-errors
     generate_docs.commands += $$doc_command -generate $$DOC_INDEXES
-    prepare_docs.depends += qtattributionsscanner
 } else {
     html_docs.commands += $$doc_command $$DOC_INDEXES
-    html_docs.depends += qtattributionsscanner
 }
 
 !build_online_docs {
